packages/react/src/components/Toast/ToastProvider.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { PropsWithChildren, useContext, useMemo, useRef } from "react";
+import { Toast, ToastProps } from ":/components/Toast/index";
+import { tokens } from ":/cunningham-tokens";
+import { Queue } from ":/utils/Queue";
+
+export enum ToastType {
+  INFO = "info",
+  SUCCESS = "success",
+  WARNING = "warning",
+  ERROR = "error",
+  NEUTRAL = "neutral",
```

Comment:
I wonder if the type neutral is relevant. The neutral style could be applied when no type is provided ?

Reply:
I prefer having an enum to be exhaustive, so there's less ambiguity, WDYT?

Reply:
Right !